patches/eslint-import-resolver-typescript+2.5.0.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/node_modules/eslint-import-resolver-typescript/lib/cjs.js b/node_modules/eslint-import-resolver-typescript/lib/cjs.js
+index 5aeddb5..963433c 100644
+--- a/node_modules/eslint-import-resolver-typescript/lib/cjs.js
++++ b/node_modules/eslint-import-resolver-typescript/lib/cjs.js
+@@ -49,13 +49,13 @@ function __spreadArray(to, from) {
+ 
+ var IMPORTER_NAME = 'eslint-import-resolver-typescript';
+ var log = debug__default['default'](IMPORTER_NAME);
+-var defaultExtensions = __spreadArray(__spreadArray([
++var defaultExtensions = [
+     '.ts',
+     '.tsx',
+-    '.d.ts'
+-], Object.keys(require.extensions)), [
```

Comment:
When I opened this file, I got an ESLint warning that `require.extensions` was deprecated. When I ran `yarn lavamoat:auto`, then `require.extensions` was undefined. I am not sure what exactly the correct fix here is, but I don't think we need to make this complicated. The list of extensions we want to support are JavaScript and TypeScript.